spec/models/category_spec.rb

```diff
@@ -3,4 +3,49 @@
 describe Category, type: :model do
   it { should have_many(:videos) }
   it { should validate_presence_of(:name) }
+
+  describe "#recent_videos" do
+    before :each do
+      @test_category = Category.create(name: "drama")
```

Comment:
Using instance variables is a faux-pas in RSpec tests, as the instance is shared among different specs. You should use `let` or `let!`. Each spec must be isolated test-case.